ui/src/assets/chat_page.scss

```diff
@@ -0,0 +1,110 @@
+/* Make the main container a flex column that fills the available height */
+.chat-container {
+  display: flex;
+  flex-direction: column;
+  height: 95vh;
+  /* 95 viewport height */
+  width: 100%;
+  /* Full viewport width */
+  top: 0;
+  background-color: #f7f7f7;
```

Comment:
Please some specify background colors unless absolutely necessary - I'm currently going through the UI and making sure theming is possible so the fewer colors are present the better